tests/locust/tests/capacity_test.py

```diff
@@ -0,0 +1,154 @@
+import csv
+import os
+import time
+import string
+import uuid
+from locust import HttpUser
+from locust import TaskSet
+from locust import task
+from locust import constant
+from locust import LoadTestShape
+from locust import between
+from tests.lib.authentication import Authentication
+
+
+class UserTasks(TaskSet):
+    @task
+    def post_to_api(self):
+        # call_time = None
+        row = 1
+        Bearer_Token = None
+        messageBatchReference = str(uuid.uuid1())
+        routingPlanId = "119bdd50-783c-4161-a765-792785e46851"
+        while True:
+            X_Correlation_ID = str(uuid.uuid1())
+            messageReference = str(uuid.uuid1())
+
+            data = None
+            with open("tests/locust/test_data/pds-data.csv", "r", encoding="utf-8-sig") as f:
+                reader = csv.reader(f)
+                data = [[c.replace('\ufeff', '') for c in row] for row in reader]
+
+            if data is None:
+                return
+
+            # skip first line as it has headers
+            for row in range(1, len(data)):
+                nhsNumber = data[row][0]
+                # dateOfBirth = data[row][1]
+
+                Bearer_Token = Authentication.generate_int_authentication()
+
+                headers = {
+                    "accept": "application/vnd.api+json",
+                    "X-Correlation-ID": X_Correlation_ID,
+                    "Content-Type": "application/vnd.api+json",
+                    "Authorization": Bearer_Token
+                }
+
+                json = {

```

Comment:
you could either implement new static methods in the tests/lib/generators.py class(es) or move this into a shared location so we don't have duplication